chromadb/api/__init__.py

```diff
@@ -291,3 +293,8 @@ def create_index(self, collection_name: Optional[str] = None) -> bool:
 
         """
         pass
+
+    @abstractmethod
+    def persist(self):
```

Comment:
This method exists in both local.py and fastapi.py implementations but is not exposed. It should be.